### ISSUE TITLE
call to s:delimiterMap

### ISSUE THREAD

**Issue by juanpabloaj** [scrooloose/nerdcommenter#47]
How I can call to s:delimiterMap from other functions? 
// for not re write a similar dic with comments chars

**Comment by alerque** [comment]
Is this something that access to the `g:NERDCustomDelimiters` array doesn't handle? For example in your rc file you can override one set of delimiters like this:

```
let g:NERDCustomDelimiters = { 'pascal': { 'left': '(*','right': '*)' } }
```

@tiesmaster or @juanpabloaj, is there some use case that this functionality doesn't allow for?

**Comment by alerque** [comment]
Nevermind, I think I get it. You're trying to get data _out_ of the plugin, not add or manipulate what's _in_ the plugin.

**Comment by alerque** [comment]
This appears to have been fixed in 6549cfde so the map is globally accessible. If there is anything else left to do please comment back.
